R/uniDimBayesianCalcCoeff.R

```diff
@@ -622,8 +660,99 @@
     stateContainer[["itemResultsObj"]] <- createJaspState(out, dependencies = c("omegaItem",  "alphaItem",
                                                                                 "lambda2Item",  "lambda6Item",
                                                                                 "glbItem","credibleIntervalValueItem",
-                                                                                "itemRestCor", "meanItem", "sdItem"))
+                                                                                "itemRestCor", "meanItem", "sdItem",
+                                                                                "iwScale", "iwDf", "igShape", "igScale",
+                                                                                "stdCoeffs", "pointEst", "loadMean"))
+
+  }
+
+  return(out)
+}
 
+
+.BayesianFitMeasures <- function(jaspResults, dataset, options, model) {
+  if (!is.null(.getStateContainerB(jaspResults)[["fitMeasuresObj"]]$object))
+    return(.getStateContainerB(jaspResults)[["fitMeasuresObj"]]$object)
```

Comment:
this function could use some references/ links to where this comes from. It's fine if you just permalink to the source code of blavaan.